src/molecules/SubMenu.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react'
+import { Box } from 'atomic-layout'
+
+import { IoIosArrowDown } from 'react-icons/io'
+import styled from 'styled-components'
+
+const StyledLi = styled.li`
+  display: flex;
+  flex-direction: column;
+`
+
+const StyledContainerLi = styled.div`
+  display: flex;
+  justify-content: space-between;
+  box-shadow: inset 0 -1px rgba(0, 0, 0, 0.15);
+
+  :hover {
+    background: rgba(0, 0, 0, 0.2);
+    box-shadow: inset 0 -1px rgba(0, 0, 0, 0);
+  }
+`
+const Link = styled.a`
+  width: 100%;
+  display: block;
+  color: #fff;
+  font-size: 15px;
+  line-height: 28px;
+  padding: 10.5px 45px 10.5px 18px;
+  text-shadow: 0 0 1px rgba(255, 255, 255, 0.1);
+  position: relative;
+  z-index: 1;
+`
+
+const Icon = styled.div`
+  color: #fff;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: 4em;
+  font-size: 12px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 2;
+  border-left: 1px solid rgba(0, 0, 0, 0.15);
+  background: rgba(0, 0, 0, 0.05);
+  font-size: 19px;
+`
+
+const Ul = styled.ul`
+  display: block;
+  padding: 0;
+`
+
+const StyledSubMenuLi = styled.li`
+  display: block;
+  list-style: none;
+  box-shadow: inset 0 -1px rgba(0, 0, 0, 0.15);
+
+  :hover {
+    background: rgba(0, 0, 0, 0.2);
+    box-shadow: inset 0 -1px rgba(0, 0, 0, 0);
+  }
+`
+
+interface Menu {
+  name: string
+  subMenu: SubMenu[]
+}
+
+interface SubMenu {
```

Comment:
I think you should export these interfaces and reuse them in `NavMenuMob` component.

```tsx
export interface Menu { ... }
export interface SubMenu { ... }
```

Please note that your `interface SubMenu` and `export const SubMenu: React.FC` have the same name (`SubMenu`). You need to rename one of them (either interface or component).